docs/features/interceptors.md

```diff
@@ -218,12 +218,14 @@ There is an experimental public API `GetInterceptorMethod(this SemanticModel, In
 
 ### User opt-in
 
-To use interceptors, the user project must specify the property `<InterceptorsPreviewNamespaces>`. This is a list of namespaces which are allowed to contain interceptors.
+To use interceptors, the user project must specify the property `<InterceptorsNamespaces>`. This is a list of namespaces which are allowed to contain interceptors.
```

Comment:
>InterceptorsNamespaces [](http://example.com/codeflow?start=66&length=22)

Should the name be "InterceptorNamespaces" rather than "Interceptors..."?

Reply:
In the issue there seemed to be agreement on the name `InterceptorsNamespaces`, i.e. the namespaces which contain interceptors. So I prefer to keep this name.